### PR TITLE
Portfolio Menu

### PR DESCRIPTION
**Pull request by zairdon20**
In the following milestone we:

- Made sure that when the user clicks (or taps) the hamburger button, the mobile menu appears.

-  Made sure that the user clicks (or taps) the close (X) button, the mobile menu disappears.

-  Made sure that when the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

-  Made sure that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,15 +11,21 @@
...
-    <header id="Portfolio">
+    <header id="Portfolio" class="parent">
         <h2 class="log"><a class="Welcome" href="#">Welcome</a></h2>
         <nav class="navbar-humberger">
-            <i class="fa fa-bars " aria-hidden="true"></i>
+            <i class="menuIcon material-icons">menu</i>
         </nav>
         <ul class="navbar-links">
+            <li>
+            <i class="closeIcon material-icons d-none">close</i>
```
Comment (descholar-ceo) [inline index.html]: - [ ] The hamburger button and menu should follow the design from the chosen template in Figma (e.g., font, colors, images, tests, margins). => this close button should only be shown when the mobile menu is visible, kindly address this issue

- [ ] [Optional] This is just an optional requirement, don't take it hard you can decide to implement it depending on whether you have bandwidth for it or not: the mobile menu should disappear on desktop screens, currently when the screen grows, it remains in the screen. 

Kindly watch the following screencast for learning more context of my two comments above

https://user-images.githubusercontent.com/56886064/145965795-382f13fb-f9da-44d4-86e4-f61402e5b333.mp4

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @zairdon20 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (zairdon20) [reply index.html]: thanks for this feedback

**Review by alonsofalconi — APPROVED** [review]
Hi @zairdon20 and @tresorsawasawa,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**